aten/src/ATen/native/quantized/qlinear_unpack.cpp

```diff
@@ -68,6 +70,7 @@ TORCH_LIBRARY_IMPL(quantized, CPU, m) {
 }
```

Comment:
and register_linear_params() for TORCH_LIBRARY_IMPL(quantized, CPU, m) ?

Reply:
Not needed here because the linear params class is not used.